ConanInstall.cmake

```diff
@@ -38,6 +38,8 @@ if(NOT CONAN_OPENSTUDIO_ALREADY_RUN)
   # set(BOOST_VERSION "1.71.0")
 
   list(APPEND CONAN_OPTIONS "zlib:minizip=True")
+  list(APPEND CONAN_OPTIONS "fmt:header_only=True")
```

Comment:
Recommend that we remove this line and just use it as a proper library. It's much better compile times.

Reply:
If we cannot get {fmt} linking without this, then keep it in place. It's at least only one thing that has to be changed in the future.